shuffle/app/(frontend)/layout.tsx

```diff
@@ -28,11 +28,12 @@ const RootLayout = ({ children }: { children: React.ReactNode }) => (
     <html lang="en">
       <body>
         <Contexts>
-          <Header />
-          {children}
+          <LogrocketWrapper>
```

Comment:
I would move it above Contexts so we get logs if there's any errors there.

Reply:
done